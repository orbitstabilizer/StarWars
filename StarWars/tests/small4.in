4
Sector-1 SEPARATISTS
Sector-2 REPUBLIC
Sector-3 REPUBLIC
Sector-4 REPUBLIC
51
Jedi Jedi-1 33 91 154
Jedi Jedi-2 107 55 108
Jedi Jedi-3 668 74 45
Officer Officer-4 PILOTING 4
Sith Sith-5 532 79 156
Sith Sith-6 915 36 88
Sith Sith-7 456 63 22
Sith Sith-8 531 57 198
Sith Sith-9 624 38 146
Sith Sith-10 145 34 120
Sith Sith-11 211 46 65
Sith Sith-12 229 21 156
Officer Officer-13 PILOTING 4
Officer Officer-14 ENGINEERING 7
Sith Sith-15 81 38 165
Sith Sith-16 442 73 96
Sith Sith-17 374 30 121
Sith Sith-18 274 59 178
Sith Sith-19 478 99 110
Sith Sith-20 679 24 104
Sith Sith-21 902 40 49
Sith Sith-22 622 72 103
Sith Sith-23 807 73 194
Officer Officer-24 TACTICAL 3
Officer Officer-25 GUNNERY 4
Officer Officer-26 COMMAND 6
Officer Officer-27 COMMAND 9
Jedi Jedi-28 359 23 93
Jedi Jedi-29 991 12 175
Jedi Jedi-30 49 46 73
Sith Sith-31 573 95 16
Sith Sith-32 304 7 78
Sith Sith-33 419 80 197
Officer Officer-34 COMMAND 10
Officer Officer-35 GUNNERY 4
Officer Officer-36 PILOTING 1
Officer Officer-37 PILOTING 8
Jedi Jedi-38 416 16 16
Jedi Jedi-39 522 40 30
Jedi Jedi-40 578 67 17
Jedi Jedi-41 934 83 180
Jedi Jedi-42 985 86 0
Officer Officer-43 ENGINEERING 6
Officer Officer-44 GUNNERY 9
Jedi Jedi-45 99 2 13
Officer Officer-46 COMMAND 6
Officer Officer-47 PILOTING 10
Jedi Jedi-48 44 96 108
Jedi Jedi-49 229 40 92
Jedi Jedi-50 42 77 56
Officer Officer-51 PILOTING 10
8
RepublicCruiser Cruiser-1 4 2
4 1 2 3 4
SeparatistDestroyer Destroyer-2 2 3
7 5 6 7 8 9 10 11
SeparatistFrigate Destroyer-3 2 4
3 12 13 14
SeparatistBattleship Destroyer-4 1 5
7 15 16 17 18 19 20 21
SeparatistBattleship Destroyer-5 4 6
6 22 23 24 25 26 27
RepublicCruiser Cruiser-6 3 7
3 28 29 30
SeparatistBattleship Destroyer-7 1 8
7 31 32 33 34 35 36 37
RepublicCruiser Cruiser-8 3 9
4 38 39 40 41
20
20 5 1 10
20 7 4 11
50 35
50 26
50 35
30 6
11 4
51 1 Armament 6966
41 31 3
50 13
50 26
40 31 4
50 24
40 44 3
11 1
20 6 4 12
30 8
51 8 Shield 4662
51 1 Armament 4676
51 2 Shield 9808
