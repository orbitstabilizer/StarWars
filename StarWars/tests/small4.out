Warship Cruiser-1 in (Sector-4, 2)
Jedi-3 40866
Warship Destroyer-2 in (Sector-2, 3)
Sith-6 31140
Warship Cruiser-8 in (Sector-3, 9)
Jedi-41 24522
Warship Destroyer-4 in (Sector-1, 5)
Sith-21 18318
Warship Destroyer-5 in (Sector-1, 10)
Sith-23 9468
Warship Destroyer-7 in (Sector-4, 11)
Sith-31 7216
Warship Cruiser-6 in (Sector-4, 12)
Jedi-29 6552
Warship Destroyer-3 in (Sector-2, 4)
Sith-12 2338
Jedi Jedi-41 is in Cruiser-8
1363
Sith Sith-23 is in Destroyer-5
1293
Jedi Jedi-42 is free
1243
Jedi Jedi-29 is in Cruiser-6
1202
Sith Sith-6 is in Destroyer-2
1147
Sith Sith-21 is in Destroyer-4
1111
Sith Sith-22 is in Destroyer-5
1013
Sith Sith-5 is in Destroyer-2
1004
Sith Sith-19 is in Destroyer-4
984
Sith Sith-31 is in Destroyer-7
969
Sith Sith-8 is in Destroyer-2
957
Sith Sith-33 is in Destroyer-7
936
Jedi Jedi-3 is in Cruiser-1
935
Sith Sith-9 is in Destroyer-2
922
Sith Sith-20 is in Destroyer-4
879
Sith Sith-16 is in Destroyer-4
830
Jedi Jedi-40 is in Cruiser-8
796
Sith Sith-7 is in Destroyer-2
730
Sith Sith-18 is in Destroyer-4
688
Jedi Jedi-39 is in Cruiser-8
672
Sith Sith-17 is in Destroyer-4
615
Jedi Jedi-28 is in Cruiser-6
521
Jedi Jedi-38 is in Cruiser-8
480
Sith Sith-12 is in Destroyer-3
469
Jedi Jedi-1 is in Cruiser-1
460
Sith Sith-11 is in Destroyer-2
460
Jedi Jedi-49 is free
441
Jedi Jedi-48 is free
440
Sith Sith-32 is in Destroyer-7
410
Sith Sith-10 is in Destroyer-2
401
Sith Sith-15 is in Destroyer-4
398
Jedi Jedi-2 is in Cruiser-1
380
Jedi Jedi-50 is free
329
Jedi Jedi-30 is in Cruiser-6
260
Jedi Jedi-45 is free
118
Officer Officer-34 is in Destroyer-7
COMMAND 10
Officer Officer-47 is free
PILOTING 10
Officer Officer-51 is free
PILOTING 10
Officer Officer-27 is in Destroyer-5
COMMAND 9
Officer Officer-44 is in Destroyer-3
GUNNERY 9
Officer Officer-26 is in Destroyer-5
COMMAND 8
Officer Officer-37 is in Destroyer-7
PILOTING 8
Officer Officer-14 is in Destroyer-3
ENGINEERING 7
Officer Officer-35 is in Destroyer-7
GUNNERY 6
Officer Officer-43 is free
ENGINEERING 6
Officer Officer-46 is free
COMMAND 6
Officer Officer-13 is in Destroyer-3
PILOTING 5
Officer Officer-4 is in Cruiser-1
PILOTING 4
Officer Officer-24 is in Destroyer-5
TACTICAL 4
Officer Officer-25 is in Destroyer-5
GUNNERY 4
Officer Officer-36 is in Destroyer-7
PILOTING 1
